core/src/main/java/com/airbnb/aerosolve/core/models/AdditiveModel.java

```diff
@@ -294,4 +294,8 @@ public void updateDense(float gradWithLearningRate,
       }
     }
   }
+
+  public void resample(int newBins) {
```

Comment:
one idea is Mutable Model used in Training, and Generate Immutable model from Training. the Mutable model has resample method, while immutable model doesn't